doc/source/notebooks/gpflow2_upgrade_guide.md

```diff
@@ -61,8 +76,16 @@ Usage of GPflow’s Scipy optimizer has changed. It has been renamed from `gpflo
  * The options (`disp`, `maxiter`) must now be passed in a dictionary.
 
 For example:
-
-![comparison of how to use Scipy optimizer](scipy_optimizer.png)
+```diff
+-optimizer = gpflow.train.ScipyOptimizer()
+-optimizer.minimize(model, disp=True, maxiter=100)
++optimizer = gpflow.optimizers.Scipy()
++optimizer.minimize(
++    lambda: -model.log_marginal_likelihood(),
```

Comment:
do we want to mention `tf.function` and jit here?

Reply:
Scipy.minimize has `jit=True` by default